Reference-Platform/Platforms/Enterprise/ReleaseNotes-17.12.md

```diff
@@ -57,7 +57,7 @@ The goal of the Linaro Enterprise Reference Platform is to provide a fully teste
 - [3499](https://bugs.linaro.org/show_bug.cgi?id=3499) [HP-m400] CPU\*: failed to boot: -6
 
 ### DPDK is released with the following known bugs:
-- [3169](https://bugs.linaro.org/show_bug.cgi?id=3169) DPDK: can't enable sr-iov on D05 with Intel 82599
+- [3169](https://bugs.linaro.org/show_bug.cgi?id=3169) DPDK: can't enable sr-iov on D05 with Intel 82599. SMMU is not enabled and SR-IOV is not enabled well on D05.
```

Comment:
Please, move this to the D05 section, rather than DPDK. Then remove DPDK from the title of the bug, since this is wider than just DPDK.